deploy/sdk/src/dynamo/sdk/cli/allocator.py

```
@@ -21,8 +21,7 @@
 import os
 from typing import Any
 
-from _bentoml_sdk import Service
-from simple_di import inject
+from dynamo.sdk.core.protocol.interface import ServiceInterface
```

Pylint (3.3.7)

[error] Unable to import 'dynamo.sdk.core.protocol.interface'

(E0401)